src/platform/packages/shared/kbn-storage-adapter/index.ts

```diff
@@ -25,12 +25,12 @@ interface StorageSchemaProperties {
   [x: string]: StorageMappingProperty;
 }
 
-export interface StorageSchema {
+export interface StorageMappings {
   properties: StorageSchemaProperties;
 }
 
 interface StorageSettingsBase {
-  schema: StorageSchema;
+  mappings: StorageMappings;
```

Comment:
Calling ES mappings "schema" has historically led to developers mapping every field. This is not necessary at best at wasteful at worst. Building inverted indices for fields you don't want to search over should generally be avoided.